backend/config/validator/validation.yaml

```diff
@@ -219,3 +219,10 @@ App\Request\Order\OrderUpdateCaptainOrderCostRequest:
             - NotBlank: ~
         captainOrderCost:
             - NotBlank: ~
+
+App\Request\Order\OrderUpdateCaptainArrivedRequest:
+    properties:
+        id:
+            - NotBlank: ~
+        isCaptainArrived:
```

Comment:
While _isCaptainArrived_ of type **bool**, _false_ value would not considered as a not-blank value by the validator, so that, either we have to change **NotBlank** to a suitable validation, or just remove the field.